TOC.md

```diff
@@ -189,7 +189,8 @@
     + [Quick Start](/get-started-with-tispark.md)
     + [User Guide](/tispark-overview.md)
 + Reference
-  + [Architecture](/architecture.md)
+  + Cluster Architecture
+    + [Overview](/architecture.md)
```

Comment:
@lilin90 @uglyengineer The corresponding Chinese file is `tidb-architecture.md`. Different file names might bring inconvenience to later maintenance. Is this by design?

Reply:
Suggest renaming it to tidb-architecture.md to align the Chinese doc. @lilin90 @uglyengineer

Reply:
Sure. Updated.